src/Helius.ts

```diff
@@ -156,8 +156,8 @@ export class Helius {
             const webhook = await this.getWebhookByID(webhookID);
             const accountAddresses = webhook.accountAddresses.concat(newAccountAddresses)
             webhook.accountAddresses = accountAddresses;
-            if (accountAddresses.length > 10000) {
-                throw new Error(`a single webhook cannot contain more than 10,000 addresses`)
+            if (accountAddresses.length > 100000) {
```

Comment:
nit: I always like writing intervals of 1000 like this: `100_000`. makes it easier to read.